arch/generic/crc32_chorba_c.c

```
@@ -1,4 +1,9 @@
 #include "zbuild.h"
+#include "arch_functions.h"
```

Cppcheck (2.19.0)

[information] Include file

(missingInclude)